package.json

```diff
@@ -41,6 +41,7 @@
     "test": "grunt"
   },
   "dependencies": {},
+
```

Comment:
Actually normalize.css should be here it is a dependency not a dev dependency but the blank line was an error.